.github/workflows/c-cpp_tmpl.yaml

```diff
@@ -28,17 +31,27 @@ jobs:
         env:
           cmake_version: ${{ matrix.cmake_version }}
           gcc_version: ${{ matrix.gcc_version }}
-        uses: NWChemEx-Project/.github/actions/get_dependencies@master
+          clang_version: ${{ matrix.clang_version }}
+        uses: NWChemEx-Project/.github/actions/get_dependencies@parallel-template
         with:
           dependencies: ${{inputs.dependencies}}
-      - name: Build and test
+      - name: Build and test using Clang
+        if: ${{inputs.clang-build == true}}
+        env:
+          cmake_version: ${{ matrix.cmake_version }}
+          clang_version: ${{ matrix.clang_version }}
+        uses: NWChemEx-Project/.github/actions/build_and_test@parallel-template
+        with: 
+          CPP_GITHUB_TOKEN: ${{ secrets.CPP_GITHUB_TOKEN }}
+      - name: Build and test using GCC
+        if: ${{inputs.clang-build == false}}
         env:
           cmake_version: ${{ matrix.cmake_version }}
           gcc_version: ${{ matrix.gcc_version }}
-        uses: NWChemEx-Project/.github/actions/build_and_test@master
+        uses: NWChemEx-Project/.github/actions/build_and_test@parallel-template
```

Comment:
master again?

Reply:
Yeah, that should be master. I will fix and push again. @ryanmrichard